source/2019-08-16-the-ember-times-issue-111.md

```diff
@@ -118,15 +118,6 @@ In the latest [episode on the Crash Log Podcast](https://crashlog.transistor.fm/
 <!-- alex ignore crash -->
 Listen to the full episode on [the Crash Log Podcast website](https://crashlog.transistor.fm/episodes/episode-6)!
 
-
```

Comment:
@jessica-jordan let's add the rule to ignore, if it was causing issue. what rule was this? https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md012?

Reply:
Yes, exactly that's the one:

```
fatal: remote base already exists.
source/2019-08-16-the-ember-times-issue-111.md: 121: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 2]
source/2019-08-16-the-ember-times-issue-111.md: 122: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 3]
source/2019-08-16-the-ember-times-issue-111.md: 123: MD012/no-multiple-blanks Multiple consecutive blank lines [Expected: 1; Actual: 4]
The command "bash scripts/markdown-lint.sh" exited with 1.
```

I can update the `markdownlint` config accordingly :+1:

Reply:
nice, thanks! sorry it's been a bit wonky, def don't want to introduce blockers